components/list/list-item-drag-drop-mixin.js

```diff
@@ -632,6 +634,7 @@ export const ListItemDragDropMixin = superclass => class extends superclass {
 		} else {
 			if (this.shadowRoot) {
 				const nodeImage = this.shadowRoot.querySelector('.d2l-list-item-drag-image') || this;
+				nodeImage.classList.add('dragging');
```

Comment:
I added a `dragging` class and used it in `list-item-mixin` in order to set `display: none` on the add button slot when dragging. This is because the tooltip in add button is causing additional height on the drag preview. This seems to be a long existing issue in Chrome and Safari, where non-visible children can affect the drag preview heights.

I think that even if we do have the add button always visible, it's still okay to hide it on drag and drop, since we don't show the outer borders anyways in the drag preview, and drag and drop is about moving the item itself.

I did try to use the `dragging` property but because it's set to true in `setTimeout` it doesn't trigger the css rule.

Reply:
Seems reasonable. Do we always get the extra height, or is it only when the tooltip is shown?